docs/examples/Asymmetric Calculation Example.ipynb

```diff
@@ -0,0 +1,884 @@
+    "\n",
+    "In general, the following is advised:\n",
+    "\n",
+    "* Use the non-independent batch dataset approach whenever few parameters change per scenario, but the batch samples many different components, e.g. during N-1 tests.\n",
+    "* Use the independent batch dataset approach when a dense sampling of the parameter space is desired for relatively a few different components, e.g. during time series power flow calculation\n",
+    "\n",
+    "See also [performance guide](https://power-grid-model.readthedocs.io/en/stable/user_manual/performance-guide.html) for the latest recommendations."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "7dd1f621",
+   "metadata": {},
+   "source": [
+    "#### Examples\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "93a22edb",
+   "metadata": {},
+   "source": [
+    "##### Time Series Profile\n",
+    "\n",
+    "Here we present following code creates a load profile with 10 timestamps for load_7. For N-1 Scenario we refer to `Power Flow Example`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 35,
+   "id": "42d5cd8f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# note the shape of the array, 10 scenarios, 1 objects (asymmetric load_7)\n",
+    "load_profile = initialize_array(\"update\", \"asym_load\", (10, 1))  \n",
+    "\n",
+    "# this is a scale of asym_load from 0% to 100%------------------\n",
+    "# the array is an (10, 1, 3) shape, which shows (scenario, object, phase).\n",
+    "# Users can always customize the load_profile in different ways.\n",
+    "load_profile[\"id\"] = [7]\n",
+    "load_profile[\"p_specified\"] = [10e6, 20e6 , 0] * np.linspace(0, 1, 10).reshape(-1, 1, 1)\n",
+    "\n",
+    "time_series_mutation = {\"asym_load\": load_profile}"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "363aa67d",
+   "metadata": {},
+   "source": [
+    "We can calculate the time series and print the current of the lines."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "id": "4e25006f",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Validation (optional)\n",
+    "from power_grid_model.validation import assert_valid_batch_data\n",
+    "assert_valid_batch_data(input_data=asym_input_data, update_data=time_series_mutation, symmetric=False, calculation_type=CalculationType.power_flow)\n",
+    "\n",
+    "# Batch Asymmetric Power Flow Calculation\n",
+    "output_data = asym_model.calculate_power_flow(update_data=time_series_mutation, symmetric=False)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "c9d23a92",
+   "metadata": {},
+   "source": [
+    "###### Accessing batch data\n",
```

Comment:
Consider collapsing the three layer "Examples/Time Series Profile/Accessing batch data" header to "Time Series Profile Example/Accessing batch data". The last one was too deep in the hierarchy and almost not legible.